src/core/components/ButtonIcon.tsx

```diff
@@ -12,6 +12,7 @@ interface TooltipProps {
   text: string;
   position?: TooltipPosition;
   delay?: number;
+  backgroundColor?: string;
```

Comment:
In order to render a tooltip that contrasted with the background, I needed to pass in a custom color

![Screenshot 2025-03-04 at 4 01 33 PM](https://github.com/user-attachments/assets/a46603f1-9653-4b55-bb60-2c418aae4389)